src/Angor/Shared/Models/AccountInfo.cs

```diff
@@ -11,6 +11,9 @@ public class AccountInfo
     public List<AddressInfo> AddressesInfo { get; set; } = new();
     public List<AddressInfo> ChangeAddressesInfo { get; set; } = new();
 
+    public List<UtxoData> PendingAdd { get; set; } = new();
+    public List<UtxoData> PendingRemove { get; set; } = new();
```

Comment:
We duplicate the data in storage, can't we just set it as a property on the UTXO data that is held in the address info utxo data?

Reply:
I tried that but it go more complicated, the duplication is negligible anyway

Reply:
Because of duplicates you need to pass both objects to the wallet operations and check each output you try to use to make sure it's not already used, this seems like you try to avoid a problem only to get it again down the line.

Reply:
The issue is more complicated, if we start to change the state that we receive form the indexer then I will have to look at the state when syncing, the proposed flag on the object will get overridden when we refresh but the utxo is not confirmed yet, though we could overcome that by changing the structure of how we store data.